### DISCUSSION TITLE
`custom_section` placement

### DISCUSSION THREAD

**Discussion by olimorris** [utilyre/barbecue.nvim#66] [Ideas]
It would be great to have the option for the `custom_section` to be placed at the start or the end. I like to display the little vim icon in my winbar:

![Screen Shot 2023-02-11 at 12 29 08@2x](https://user-images.githubusercontent.com/9512444/218258006-6fc8e75d-eabf-455a-84a9-b38c98fb5d6e.png)

But it would look a whole lot better (imo) at the start rather than the end.

**Comment by utilyre** [comment]
Hi, see 1e2048b.
